Bring changes from rc2-prep back to develop branch (#274)
JoshOrndorff/recipes: text/2-appetizers/4-events.md

To declare an event, use the
[`decl_event!` macro](https://crates.parity.io/frame_support/macro.decl_event.html). Like any rust

> enum, Events have names and can optionally carry data with them. The syntax is slightly different

Reviewer: ```suggestion
enum, Events have names and may carry optional data. The syntax is slightly different
```
